coriolis/tests/api/v1/test_utils.py

```diff
@@ -0,0 +1,239 @@
+# Copyright 2023 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+
+import ddt
+from webob import exc
+
+from coriolis.api.v1 import utils
+from coriolis import exception
+from coriolis import schemas
+from coriolis.tests import test_base
+
+
+@ddt.ddt
+class UtilsTestCase(test_base.CoriolisBaseTestCase):
+    """Test suite for the Coriolis Utils v1 API"""
+
+    def test_get_show_deleted(
+        self
+    ):
+        val = """true"""
```

Comment:
Please remove the extra string quotes. These triple-quotes are usually used for multi-line strings.
```suggestion
        val = "true"
```